app/src/main/java/sdp/moneyrun/ui/game/EndGameActivity.java

```diff
@@ -61,8 +67,20 @@ public void onClick(View v) {
 
     }
 
+//    /**
```

Comment:
If it's for later use add a todo otherwise, I don't think there should be commented code

Reply:
Agreed